go.mod

```diff
@@ -3,16 +3,23 @@ module github.com/minio/minio-go/v7
 go 1.12
 
 require (
-	github.com/dustin/go-humanize v1.0.0 // indirect
 	github.com/google/uuid v1.1.1
 	github.com/json-iterator/go v1.1.10
+	github.com/klauspost/cpuid v1.3.1 // indirect
```

Comment:
none of these are used by `minio-go` I am not able to see where is this coming from?

Reply:
it is coming from the md5-simd dependency